enterprise/frontend/src/metabase-enterprise/caching/components/StrategyFormLauncher.styled.tsx

```diff
@@ -20,9 +20,9 @@ export const PolicyToken = styled(Button)<
   justify-content: center;
   ${({ variant }) =>
     css`
-      border-color: ${color(
-        ["filled", "outline"].includes(variant || "") ? "brand" : "border",
-      )} !important;
+      border-color: ${["filled", "outline"].includes(variant || "")
+        ? color("brand")
```

Comment:
shall we use `var(--mb-color-brand)` here too, so it's consistent with `border` below?